test/runtests.jl

```diff
@@ -35,6 +35,7 @@ using DynamicPPL: getval, getlogp
 using ForwardDiff: Dual
 using MCMCChains: Chains
 using StatsFuns: binomlogpdf, logistic, logsumexp
+using TimerOutputs: TimerOutput, @timeit, print_timer
```

Comment:
It doesn't matter but sometimes I use as rule of thumb to only load stuff that I call >= 10 times, or not even in these cases. Since `TimerOutput` and `print_timer` are only used once, we could eg only import
```suggestion
using TimerOutputs: TimerOutputs, @timeit
```
In any case, it seems `print_timer` and `show` are the same (apart from a newline?), so it's not needed:
```suggestion
using TimerOutputs: TimerOutput, @timeit
```